packages/block-library/src/post-title/block.json

```diff
@@ -5,6 +5,15 @@
 		"postId",
 		"postType"
 	],
+	"attributes": {
+		"align": {
+			"type": "string"
+		},
+		"level": {
+			"type": "number",
+			"default": 2
+		}
+	},
```

Comment:
See the Heading block for how to handle this in a way compatible with global styles.

We need to use `__experimentalSelector`.